Team EPYC Delivery

manishpandeyprojects opened the pull request:
- Swap the font to **dm mono** globally
- Closes #14  **- Done**
- Created the "About Us" page based on the Figma design.
-  Closes #13  **- Done**
- Renamed the "Resources" page it to "Publications."
- Added a "Press" section on the Publications page, using the same format as the "TALKS" section. Closes #12  **- Done**
- Closed #15  **- Done**

guelowrd commented:
**General remarks**
- Navigation bar should not include CAREERS (Careers page is only accessible from the ABOUT page basically)
- The footer has not been updated (background should have this very light gray color as per the Figma)
_Reminder note to our ourselves: we need to add a link to the Roadmap page somewhere._

**Homepage**
- Need more space between "Our investors" and the mail collection banner (it should be the same space than between the end of "The way blockchains were always meant to work" paragraph and the top on "Our investors" section)

**Publications**
- The order in the left side nav bar is off ("Talks" and "Research Papers" need to be swapped) – it should fit the order in which links are displayed on the page i.e.:
- - Press
- - Blog
- - Talks
- - Research Papers 
- You mentioned you added a system for metadata for blog posts (MDX files) but it seems `app/content/resource.blog.privacy.mdx` hasn't been updated – not necessarily a problem as I can do it myself but wanted to be sure that the system consists in specifying the Title and Description in the MDX header
_Reminder note to ourselves: we need to update the list of the Press articles and their links, as they are all heading to https://miden.xyz/ at the moment._

**Careers**
- Can we add a highlight upon hovering so that it's clear links are clickable (e.g. [MIDEN DEVELOPER RELATIONS ENGINEER])? We can use the same hovering than we have in PUBLICATIONS page
_Reminder note to ourselves: we need to update the list of open roles and their details._

manishpandeyprojects commented:
I have implemented all the comments added above in the same PR and merged the Latest code from 0xMiden/homepage.

✅ **Nav** : Removed CAREERS from the top nav; it’s only reachable from About now.
✅ **Footer** : Updated background to the very light gray as per Figma.
✅ **Homepage** : Increased space between Our investors and the newsletter banner to match the spacing above Our investors.
✅ **Publications** :  Left nav order: Now Press → Blog → Talks → Research Papers.
✅ **Careers** : Added hover highlight on role links, matching Publications.

**Blog Posts**:  app/content/resource.blog.privacy.mdx - I didn’t change this file because it already has the Title and Description in the MDX header. You can check my previous commit here [[Link](https://github.com/0xMiden/homepage/pull/26/commits/c48ed08ad3c919a5571ff22fa0190c25032a2c94#diff-3a495b4442651d9916e14049fc7fd83beedc0cee58240ad02ad515a0f5bc35cd)] . do you need the different Title and Description here?

guelowrd commented:
Everything looks good except this minor thing – the gray line should be removed for the footer now that the delimitation is done with the background color.
Said differently – the line in the yellow rectangle should be removed
<img width="851" height="603" alt="Screenshot 2025-08-27 at 22 23 05" src="https://github.com/user-attachments/assets/29ab001a-2536-4c41-9c3b-a999cdf1d6b4" />

manishpandeyprojects commented:
**Roadmap Page:**

- ✅ Updated the content.
- ✅ Made the version text unclickable on both mobile and desktop.
- ✅ Added X-axis edges to the roadmap grid.
- ✅ Left-aligned the bullet points as suggested.

**All Pages:**

- ✅ Set the font size to 28px on all pages.

**Home Page:**

- ✅ Made the heading uppercase as suggested.
